mcp-server/src/mcpServerApplication.ts

```diff
@@ -108,68 +104,35 @@ export class McpServerApplication {
 
```

Comment:
Updated according to the latest TypeScript SDK's README file, with simpler code.